### DISCUSSION TITLE
Github Education expired before deadline

### DISCUSSION THREAD

**Discussion by blaycoder** [community/community#137340] [GitHub Education]
### Hi everyone,

I am confused as to why my GitHub Education account didn't last a year or even six months before I received an expiration notification.
Kindly help me look into the issue.
Thank you

**Comment by AkshayMoolya** [comment]
It's unusual for a GitHub Education account to expire before the expected duration. Here are some possible reasons and steps to address the issue:

1. **Verification Issues**: GitHub may require periodic verification of student status.
   - Check your email for any verification requests.
   - Ensure your school email is still active and linked to your GitHub account.

2. **Change in Eligibility**: Your status might have changed.
   - Confirm you still meet the eligibility criteria for GitHub Education.

3. **Technical Glitch**: There could be a system error.
   - Try logging out and back into your GitHub account.
   - Clear your browser cache and cookies.

4. **Misunderstanding of Terms**: Review the terms of your specific GitHub Education offer.
   - Some programs might have shorter durations or different conditions.

5. **Contact GitHub Support**: If the issue persists:
   - Reach out to GitHub Education support directly.
   - Provide details about your account and the unexpected expiration.

6. **Check Spam Folder**: Look for any missed communications from GitHub.
   - Important notices might have been filtered as spam.

Remember to have your student ID and relevant documentation ready when contacting GitHub support. They should be able to provide specific information about your account status and help resolve any issues.
